### DISCUSSION TITLE
Freesurfer Longitudinal Stream Error

### DISCUSSION THREAD

**Discussion by asw39** [brainvisa/brainvisa.github.io#97] [Q&A]
Hi,

This is my first time seeking support here so I apologize if I am not familiar with the community's conventions.

I am encountering errors when processing Freesurfer longitudinal stream (Freesurfer v6.0). Cross-sectional streams proceeded and completed without error, but base template creation and all following steps failed.

**Created a txt script for the base template creation: Here is a sample:**
module load freesurfer
export FREESURFER_HOME=/.../.../freesurfer
source $FREESURFER_HOME/SetUpFreeSurfer.sh

recon-all -base SA14c -tp P10 -tp P11 -tp P12 -all

**This script was separate from the following step (this was a separate .txt file):**
recon-all -long P12 SA14c -all -no-isrunning
recon-all -long P12 SA14c -brainstem-structures

**The common error I received from the base template creation: (from the recon-all.error log)**
$Id: recon-all,v 1.580.2.16 2017/01/18 14:11:24 user Exp $
/.../.../freesurfer/bin/recon-all
PWD /.../.../subjects/SA14c/label
CMD mri_label2label --srcsubject fsaverage --srclabel /.../.../subjects/fsaverage/label/lh.BA1_exvivo.label --trgsubject SA14c --trglabel ./lh.BA1_exvivo.label --hemi lh --regmethod surface

**Following the template creation, it was not possible to go from there to create the individual longitudinal mri output based on the base template**

I'd be really grateful for help with this debugging! Thank you.

**Comment by denisri** [comment]
Hi,
I'm afraid you should rather ask to a Freesurfer-centered community: here it is a Brainvisa discussion forum, and we have a limited expertise with Freesurfer, even if one of the Brainvisa modules do wrap some calls to Freesurfer.
Moreover I don't understand your problem: I see no error message in your message and log.
